src/main/java/com/algorand/algosdk/abi/ABIType.java

```diff
@@ -118,8 +118,13 @@ else if (str.charAt(i) == ')') {
                 if (parenStack.isEmpty())
```

Comment:
similar error in the exception above in 106

Reply:
👍 good find